.circleci/config.yml

```diff
@@ -25,10 +25,10 @@ aliases:
     - run:
         name: "Import signing key"
         command: |
+          gpg --keyserver keyserver.ubuntu.com \
```

Comment:
It's worth surfacing that we're specifying the Ubuntu keyserver here to mirror the [approach taken](https://github.com/jodersky/sbt-gpg/blob/master/.ci/build) by @jodersky. Through testing, I've found that this keyserver was updated more quickly than the default.